index.html

```diff
@@ -301,19 +362,68 @@ <h1>Contact Us</h1>
                 </div>
 
                 <div class="email-input">
-                  <input name="Email" id="email" type="email" required placeholder="jeremy@gmail.com"><br>
-                  </div> 
-                
+                  <input name="Email" id="email2" type="email" required placeholder="jeremy@gmail.com"><br>
+                </div>
+
               </div>
 
               <div class="text-area">
                 <textarea maxlength="500" placeholder="Enter text here"></textarea>
               </div>
 
+
               <div class="contact-btn">
                 <button class="button button-1" type="submit">See Project</button>
               </div>
 
+
+            </form>
+
+          </div>
+
+        </div>
+
+      </div>
+
+
+      <div class="container4">
+
+        <div class="contact-us2">
+
+          <div class="contact-header">
+            <h1>Let's chat</h1>
+          </div>
+
+          <div class="contact-paragraph">
+            <p>If you have an application you are interested in developing, a feature that you need built or a project
+              that needs coding. I’d love to help with it </p>
+          </div>
+
+          <div class="form1">
+
+            <form action="https://formspree.io/f/form_id" method="post">
+
+              <div class="text-field">
+
+                <div class="name-input">
+                  <input type="text" name="text-input" maxlength="30" required placeholder="Jeremy">
+                  <input type="text" name="text-input" id="lname" maxlength="30" required placeholder="Last name">
+                </div>
+
+                <div class="email-input">
+                  <input name="Email" id="email" type="email" required placeholder="jeremy@gmail.com"><br>
+                </div>
+
+              </div>
+
+              <div class="text-area">
+                <textarea maxlength="500" placeholder="Enter text here"></textarea>
+              </div>
+
+              <div class="contact-btn">
+                <button class="button button-1" type="submit">Send</button>
+              </div>
```

Comment:
- [ ] As requested by the previous reviewer, Please fix the width of the submit button, use the correct background color and make sure it matches the design on [the template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A10).

### Your design

![image](https://user-images.githubusercontent.com/83803414/159977630-19dcbdfe-ffd6-47f4-a4c0-f6b716e3407b.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/159977707-1c5e7f9b-c6c7-4f74-b12e-0d9ccc5efe62.png)